components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/LegacyJobOrchestrator.cs

```diff
@@ -495,11 +496,12 @@ public class LegacyJobOrchestrator : IJobOrchestrator, IPublisherConfigServices,
                     }
                 }
                 if (!found) {
-                    throw new MethodCallStatusException((int)HttpStatusCode.NotFound, "Endpoint not found.");
+                    throw new MethodCallStatusException((int)HttpStatusCode.NotFound, $"Endpoint: {request.EndpointUrl} not found.");
                 }
 
                 // fire config update so that the worker supervisor pickes up the changes ASAP
                 TriggerAgentConfigUpdate();
+                response.Add($"{nameof(PublishNodesAsync)}# Publishing succeeded for EndpointUrl: {request.EndpointUrl}");
```

Comment:
Let's not include name of the method (`nameof(PublishNodesAsync`) in response message. This is useful for logging, but for the caller it would not add much of information.

Reply:
It would also be a bit confusing as the direct method name is not `PublishNodesAsync`. This is our internal naming for the handler and should not be exposed.

Reply:
removed.